io.github.Sunderland93.sway-input-config.yml

```diff
@@ -0,0 +1,72 @@
+id: io.github.Sunderland93.sway-input-config
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm15
+command: sway-input-config
+rename-desktop-file: sway-input-config.desktop
+rename-icon: sway-input-config
+finish-args:
+- "--socket=wayland"
+- "--device=dri"
+- "--filesystem=/run/user:ro"
+- "--filesystem=xdg-config/sway"
+modules:
+- name: python3-appdirs
+  buildsystem: simple
+  build-commands:
+   - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} appdirs
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/3b/00/2344469e2084fb287c2e0b57b72910309874c3245463acd6cf5e3db69324/appdirs-1.4.4-py2.py3-none-any.whl
+    sha256: a841dacd6b99318a741b166adb07e19ee71a274450e68237b4650ca1055ab128
+- name: python3-i3ipc
+  buildsystem: simple
+  build-commands:
+   - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} i3ipc
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/de/30/294b07ddeccb58855c890c3ef3a951c3b0c1e2d089666d548b6a9edc39fb/i3ipc-2.2.1-py3-none-any.whl
+    sha256: c0b898223d50d42c90c818deb5033d1304c582755547dee7d15df3e3781bc690
+  - type: file
+    url: https://files.pythonhosted.org/packages/fc/b8/ff33610932e0ee81ae7f1269c890f697d56ff74b9f5b2ee5d9b7fa2c5355/python_xlib-0.33-py2.py3-none-any.whl
+    sha256: c3534038d42e0df2f1392a1b30a15a4ff5fdc2b86cfa94f072bf11b10a164398
+- name: pyside2
+  build-options:
+     prepend-path: /usr/lib/sdk/llvm15/bin
+     prepend-ld-library-path: /usr/lib/sdk/llvm11/lib
+  buildsystem: simple
```

Comment:
this should be `buildsystem: cmake-ninja` and line 41 part of the `shell` source below.